run.py

```diff
@@ -13,16 +15,24 @@ def start_server(host, port):
     app.run(debug=DEBUG, host=host, port=port, threaded=True)
 
 
+def authorize_server(id, name, token):
+    server = Server(server_id=id, server_name=name, server_token=token)
+    try:
+        db.session.add(server)
+        db.session.commit()
+    except IntegrityError:
+        db.session.rollback()
+        # Should notify the user that the server_token is already in use
```

Comment:
Something like: `app.logger.error('Server authentication token must be unique')`